packages/cosmic-swingset/test/unitTests/test-lib-wallet.js

```diff
@@ -0,0 +1,267 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from 'tape-promise/tape';
+import bundleSource from '@agoric/bundle-source';
+
+import { E } from '@agoric/eventual-send';
+import produceIssuer from '@agoric/ertp';
+import { makeZoe } from '@agoric/zoe';
+import { makeRegistrar } from '@agoric/registrar';
+import harden from '@agoric/harden';
+import { makeGetInstanceHandle } from '@agoric/zoe/src/clientSupport';
+
+import { makeWallet } from '../../lib/ag-solo/vats/lib-wallet';
+
+const setupTest = async () => {
+  const contractRoot = `${__dirname}/automaticRefund`;
```

Comment:
As you asked, the answer is "yes".  You can use `require.resolve('@agoric/zoe/src/contracts/automaticRefund')` to find the path to it without the trampoline module.

Reply:
Thanks! I'll do that